docs/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -180,32 +209,3 @@ sentry {
     }
 }
 ```
-
-### Enable Logging for `sentry-cli`
-
-You can also enable logging for `sentry-cli` by adding this config before the above `apply from:` line:
-
-```groovy {filename:android/app/build.gradle}
-project.ext.sentryCli = [
-    logLevel: "debug"
-]
-
-apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
-```
-
-### Fetch `sentry.properties`
-
-We also support fetching different `sentry.properties` files for different flavors. For that, you need to add:
-
-```groovy {filename:android/app/build.gradle}
-project.ext.sentryCli = [
-    logLevel: "debug",
-    flavorAware: true
-]
-
-apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
-```
-
-The corresponding flavor files should also be placed within the specific build type folder where you intend to use them. For example, the "Android demo release" flavor would be `react-native/android/sentry-demo-release.properties`.
-
-We recommend leaving `logLevel: "debug"` since we look for specific `sentry.properties` files depending on your flavor's name.
```

Comment:
did you remove this on purpose?

Reply:
Yes, its included in https://github.com/getsentry/sentry-docs/pull/11163/files#diff-d112867dfaecab66d0fcf0d26378af89f8610d07275f290819e64f96775292f7R153 and https://github.com/getsentry/sentry-docs/pull/11163/files#diff-d112867dfaecab66d0fcf0d26378af89f8610d07275f290819e64f96775292f7R155

Hopefully it will be easier to find as based on the incoming issues, the section at the end of the page was easily overlooked.